config/routes.rb

```diff
@@ -3,12 +3,13 @@
   namespace :api do
     namespace :v1 do
       get '/plants/find', to: 'plant_search#index'
-      resources :users, only: [:index, :create, :update, :destroy] do
+      resources :users, only: [:index, :create, :show, :update, :destroy] do
```

Comment:
Did we decide to change the user show endpoint? I see that you added it here, but I have it using a different URI below.